src/aiidalab_qe/app/result/components/viewer/structure/structure.py

```diff
@@ -26,11 +27,22 @@ def _render(self):
             """)
             self.atom_coordinates_table = TableWidget()
             self._generate_table(structure.get_ase())
-            self.results_container.children = [
+
+            # Basic widgets
+            children = [
                 self.widget,
                 self.table_description,
                 self.atom_coordinates_table,
             ]
+
+            # Add structure info if it is a relaxed structure
+            if "relax" in self._model.properties:
+                self._initialize_structure_info(structure)
+                children.insert(0, self.structure_info)
+
+            # Add the children to the container
+            self.results_container.children = tuple(children)
+
             self.atom_coordinates_table.observe(self._change_selection, "selected_rows")
             # Listen for changes in self.widget.displayed_selection and update the table
             self.widget.observe(self._update_table_selection, "displayed_selection")
```

Comment:
```suggestion
            self.atom_coordinates_table.observe(
                self._change_selection,
                "selected_rows",
            )
            self.widget.observe(
                self._update_table_selection,
                "displayed_selection",
            )
```

Consistent coding style 😉

Reply:
pre-commit updating this